docs/getting-started/openshift/index.md

```diff
@@ -0,0 +1,180 @@
+# ArgoCD Agent Hub and Spoke Cluster Setup Documentation Using Operator
+This document outlines the process of setting up ArgoCD Agent in a hub and spoke cluster architecture. This configuration allows a central ArgoCD instance (the hub) to manage applications deployed across multiple remote Kubernetes clusters (the spokes) through the argocd-agent.
+## Prerequisites
+Before proceeding with the setup, ensure you have the following:
+
+A running openshift/kubernetes cluster designated as the hub, with ArgoCD/openshift GitOps Operator installed.
+One or more Kubernetes clusters designated as spokes, ArgoCD/openshift GitOps Operator installed.
+oc configured to access both hub and spoke clusters.
+Operator must be installed in cluster scope mode.
+Apps in any Namespace should be enabled.
+Argocd-agentctl binary (for non-production scenario)
+
```

Comment:
`argocd-agentctl` binary must be installed.

Reply:
But we are not recommending it for production use, so should we share it as a must in documentation?

Reply:
If you use `argocd-agentctl`, the `openssl` binary is not required

Reply:
Should we mentaion either openssl or argocd-agentctl as a requirement?